src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_CAPACITY = 16;
+    public static final float LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
```

Comment:
Consider using the diamond operator to avoid raw type usage. This should be `new Node<K,V>[DEFAULT_CAPACITY];` However, due to type erasure, you'll need an unchecked cast or an alternative approach like an Array of `Object` and cast it where required.

Reply:
Changed